aries-site/src/pages/components/datatable.mdx

```diff
@@ -136,15 +138,16 @@ user's preferred analytical tool.
 
 ### Setting the width of a DataTable
 
-- Use the correct set of columns, prioritizing frequently used ones. Don't add more columns just because the data is there, as this will clutter the data table.
+- Use the correct set of columns, prioritizing frequently positioning the most important columns on the left. Don't add more columns just because the data is there, as this will clutter the data table.
```

Comment:
```suggestion
- Use the correct set of columns, positioning the most important or frequently used columns on the left. Don't add more columns just because the data is there, as this will clutter the data table.
```